obelisk/client.py

```diff
@@ -64,9 +64,12 @@ class LibbitcoinClient(ClientBase):
         'validate'
     ]
 
-    def __init__(self, address, public_key=None, log=None, heartbeat_port=9092):
+    def __init__(self, address, addresses=None, public_key=None, log=None, heartbeat_port=9092):
```

Comment:
maybe here we could combine these two arguments into one that accepts either a list or string.

And check it like:
```python
if isinstance(address, list):
    self.address = address[0]
    self.address = address
else:
    self.address = address

Also we need to think of a good way select the initial url. Probably should just be random for now, but maybe we could do it by geographic location relative to our IP in the future.